Workflow/views.py

```diff
@@ -102,21 +105,27 @@ def csv_export(request, value):
     formatted_date = datetime.now().strftime('%d-%m-%Y_%H-%M')
     csv_filename = 'email_{0}_{1}.csv'.format(value, formatted_date)
 
-    if value == 'alle':
-        users = ScholariumProfile.objects.all()
-    elif value == 'unterstuetzer':
-        users = ScholariumProfile.objects.filter(stufe__gte=1)
-    elif value == 'interessenten':
-        users = ScholariumProfile.objects.filter(stufe=0)
-
     response = HttpResponse(content_type='text/csv')
     response['Content-Disposition'] = \
-        'attachment; filename="{}"'.format(csv_filename)
+    'attachment; filename="{}"'.format(csv_filename)
 
     writer = csv.writer(response)
-    writer.writerow(['Email'])  # header-row
-
-    for one in users:
-        writer.writerow(['{}'.format(one.user.email)])
+    
+    if value == 'alle':
+        writer.writerow(['Email'])  # header-row
+        for one in ScholariumProfile.objects.all():
+            writer.writerow(['{}'.format(one.user.email)])
+    elif value == 'unterstuetzer':
+        writer.writerow(['Email'])  # header-row
+        for one in ScholariumProfile.objects.filter(stufe__gte=1):
+            writer.writerow(['{}'.format(one.user.email)])
+    elif value == 'interessenten':
+        writer.writerow(['Email'])  # header-row
+        for one in ScholariumProfile.objects.filter(stufe=0):
+            writer.writerow(['{}'.format(one.user.email)])
+    elif value == 'abgelaufen':
+        writer.writerow(['Vorname','Nachname','Email','Ablaufdatum'])  # header-row
+        for one in ScholariumProfile.objects.filter(datum_ablauf__lt=date.today()):
+            writer.writerow([one.user.first_name, one.user.last_name,one.user.email,one.datum_ablauf])
```

Comment:
Streng genommen müssten nach den Kommata in Zeile 127 & 129 jeweils Leerzeichen sein und die Zeilen 127-129 sind länger als 79 Zeichen, aber hierzu siehe obigen Kommentar.